### ISSUE TITLE
put back radio-beam conversion

### ISSUE THREAD

**Issue by jpinedaf** [kristina-monsch/ammonia_project#15]
In cell 6 of https://github.com/kristina-monsch/ammonia_project/blob/master/linefit_include22.ipynb we should also include the conversion between Jy/beam into K. 

```python
from radio_beam import Beam
beam11 = Beam.from_fits_header(fits.getheader(OneOneFile))
cube11 = pyspeckit.Cube(file=OneOneFile)
freq11= 23.6944955*u.GHz
cube11.cube *= beam11.jtok( freq11)
cube11.unit="K"
```

**Comment by jpinedaf** [comment]
not closed until NH3(2,2) is added

**Comment by kristina-monsch** [comment]
I've added the conversion now but it seems like the fitting part is not working anymore, because it gives me the following error message:
```python2.7
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-83-1d81a47e97ee> in <module>()
     24             maxpars=[0,0,17.0,0,vmax,1],
     25             position_order = 1/peaksnr, maskmap=planemask,
---> 26             multicore=2)#, start_from_point=(280,330))
     27 
     28 #save the fit parameters in a FITS-file otherwise the fit would be lost!!!

/home/monsch/Desktop/NH3/pyspeckit/pyspeckit/cubes/SpectralCube.pyc in fiteach(self, errspec, errmap, guesses, verbose, verbose_level, quiet, signal_cut, usemomentcube, blank_value, integral, direct_integral, absorption, use_nearest_as_guess, use_neighbor_as_guess, start_from_point, multicore, position_order, continuum_map, prevalidate_guesses, maskmap, **fitkwargs)
    961                 raise ex
    962         except AssertionError:
--> 963             raise AssertionError("The first pixel had the wrong fit "
    964                                  "parameter shape.  This is probably "
    965                                  "a bug; please report it.")

AssertionError: The first pixel had the wrong fit parameter shape.  This is probably a bug; please report it.
```

I tried to start the fit from another pixel but this doesn't help. Is this really a but or is there an error somewhere in my code? 
See the linefit_include22 notebook in pull request #17

**Comment by kristina-monsch** [comment]
Oh sorry, I think it works now! It seems like the xarr cannot be omitted at all, but I'll look more into that tomorrow!

**Comment by kristina-monsch** [comment]
#19 I've implemented the conversion and the fits runs now, but please have a look at the results (i.e. those with snr_min=20)! When I open the moment maps in CASA I don't see anything and I'm not sure whether this is due to an error in the code or due to something else.

**Comment by jpinedaf** [comment]
I think this is solved in the latest version, right?

**Comment by kristina-monsch** [comment]
Yep :-) but I need to compare the file we worked on last Thrusday with 
my code in the refactor_code branch before I merge them as well.

**Comment by jpinedaf** [comment]
OK, sounds good

**Comment by kristina-monsch** [comment]
this was solved in pull request #19
